src/com/cakeshop/domain/Address.java

```diff
@@ -0,0 +1,57 @@
+package com.cakeshop.domain;
+
+public class Address {
+	
```

Comment:
Add one more property in this calss that is `private String id;`
When you insert any new property in domain class, remember to
- generate `getter `and `setter` method for the property
- delete existing `toString `and re-generate it
- delete existing all argumented constructor and re-generate it